lib/core/directives/search-chip.directive.ts

```diff
@@ -0,0 +1,38 @@
+/*!
+ * @license
+ * Copyright 2019 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Directive, ElementRef, Renderer2 } from '@angular/core';
+
+@Directive({
+  // tslint:disable-next-line: directive-selector
+  selector: 'mat-chip'
+})
+export class SearchChipDirective {
+    constructor(private el: ElementRef, private renderer: Renderer2) {
+    }
+
+    ngAfterViewInit() {
+        setTimeout(() => {
+            const chipListWrapper = document.querySelector('.mat-chip-list-wrapper');
```

Comment:
this should have been left in chip-list. right now, for every chip  we will have a DOM manipulation for the same element